plugins/readtimer.koplugin/main.lua

```diff
@@ -57,6 +57,17 @@ function ReadTimer:init()
             })
         end
     end
+
+    self.update_status_bars = function()
+        if self.show_timer_value_in_header then
+            UIManager:broadcastEvent(Event:new("UpdateHeader"))
+        end
+        if self.show_timer_value_in_footer then
+            UIManager:broadcastEvent(Event:new("UpdateFooter", true))
+        end
```

Comment:
Ideally, you would not schedule this if it's doing nothing because none of the settings are enabled.